docs/setup-and-running.md

```diff
@@ -0,0 +1,102 @@
+## Setup and Running
+
+It is highly recommended to use a Unix-based system (such as Ubuntu, Fedora, Kali, or any other) or MacOS. Windows may encounter certain issues with Docker.
+
+### Requirements
+
+- Git is essential before anything else;
+- [Node][1] and [TypeScript][2];
+- If you are on MacOs, you can install above using [homebrew][3];
+- [Docker][4]
+- [pnpm][5]
+- [Prisma][6]
+- [GPG][7]
+- [PostGres][8]
+
+## 1 - Clone the Repository
+
+To clone the repository, follow these steps:
+
+Open your terminal or command prompt.
+
+Go to the file that you want to download the project (you can use the command `ls` to see where your location and the comand `cd` to go to myfiles/exemple).
+
+Then, run the following command to clone the repository:
+
+`git clone https://github.com/podcodar/app.git`
+
+This command will download the project files to your local machine in the folder `app`. ✨
+
+## 2 - Install Project Dependencies
+
+To install pnpm on your system, simply run this command if node is installed.
+
+`npm install -g pnpm`
+
+For other ways to install, follow [the official installation instructions][5] for your operating system.
+
+Then use the commands:
+
+`pnpm install`
+to install package.json dependencies
+
+`pnpm run`
+to run the script provided in package.json
+
+## 3 - Environment setup
+
+We use [GPG][9] for credential managing, first make sure you have it installed on your machine and correctly set in your path (if not, follow the requirements doc :wink:).
+
+### Generating environment variables
+
+**Installation for GPG**
```

Comment:
[Markdown lint]

Avoid using bold as title. Always try to use h1,h2,h3... blocks to define titles in your files. This enhances the final visualization, having a more distinct visualization for information blocks and groups.